sandbox/__test__/messages.spec.js

```diff
@@ -64,5 +64,87 @@ describe('/api/v1/messages', () => {
           })
           .expect("Content-Type", /json/, done);
   });
+  it('returns a 500 when the sending group has missing templates', (done) => {
```

Comment:
We also need to extend the test suite for sandbox with tests that cover these cases - these are in the `tests/sandbox` folder - @Kennedy1265 can advise on what needs to be done.

Reply:
added some tests